rts/Sim/Misc/ModInfo.h

```diff
@@ -220,6 +220,14 @@ class CModInfo
 
 	bool enableSmoothMesh;
 
+	/// Reduce the resolution of the smooth mesh by the divider value. Minimum 1, default 2.
```

Comment:
The main reason to increase this would be performance, right?

Reply:
Yup. Added comment.